website/markdown/docs/contribution/website.mdx

```diff
@@ -0,0 +1,26 @@
+---
+name: Website
+excerpt: Learn about Tuist website, how to run it locally
+---
+
+# Contributing
+
+Tuist's website is built using [Gatsby](https://www.gatsbyjs.com/).
+
+### Installing dependencies
+
+Make sure you are on the correct version of Ruby and NodeJS, which are respectively declared on the `.ruby-version` and `.nvmrc` files.
```

Comment:
I'd add that they can use a tool like chruby and nvm for that. Assuming that most contributors have a Swift background, it might not be obvious how to "choose the correct version"